<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stack tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="stackValueLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Last stack item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStackItemBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Push item to the stack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="popStackItemBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Pop item from the stack</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stackTextInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>New stack item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputLastStackItem">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Show last stack item</string>
    </property>
   </widget>
   <widget class="QLabel" name="poppedItemLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>19</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Item that was popped:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Работа со стеком</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
